tests/openwisp2/settings.py

```diff
@@ -98,6 +99,49 @@
     },
 ]
 
+CACHES = {
+    'default': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': 'redis://localhost/0',
```

Comment:
let's change this to `redis://localhost/5`, it allows us to run celery workers of different modules in development without them clashing (when all modules use 0, if you run the celery worker of the development environment of 2 different modules at the same time, they will try to load each other task, not finding them and returning errors)